### DISCUSSION TITLE
How can I add a comment to a specific paragraph or a specific selection of a text?

### DISCUSSION THREAD

**Discussion by EduNetEurope** [outline/outline#6865] [Self Hosting] [answered]
The[ help document](https://www.getoutline.com/changelog/commenting) says:

> Once enabled you'll see a new comments sidebar become available. You can add comments to any document or to a specific text selection, we also support threaded replies, notifications, @mentions, basic formatting and images in comments.

We only manage to add general comments. 
How can we add a comment to a specific text selection?

Thanks for your help.

**Comment by tommoor** [comment, marked as answer]
<img width="555" alt="image" src="https://github.com/outline/outline/assets/380914/e8bd05d3-1170-4c3c-8c5b-92d8ce1678f8">

`Cmd`+`Alt`+`m` also works as a shortcut

**Comment by EduNetEurope** [comment]
Thanks for the quick reply! Sorry for being blind.
